modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc

```diff
@@ -114,13 +120,13 @@ image::enter-timezone.png[]
 
 image::root-password.png[]
 
-. Once installation completes login as root.
+. When installation completes log in as root.
 
 . Proceed to the next section.
 
 
 
-== Register {sl-micro} and {productname} {productnumber}
+== Register {sl-micro} and {productname} {productnumber} Proxy
```

Comment:
This macro is not correct for 5.5. It's still called SLE Micro 5.5. See [download filenames](https://www.suse.com/download/sle-micro/)

Reply:
ok, using sle-micro now.